developers/weaviate/model-providers/friendliai/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: FriendliAI + Weaviate
+sidebar_position: 10
+image: og/docs/integrations/provider_integrations_friendliai.jpg
+# tags: ['model providers', 'friendliai']
+---
+
+import BetaPageNote from '../_includes/beta_pages.md';
+
+<BetaPageNote />
+
+:::info Added in `v1.27.0`
+:::
+
+FriendliAI offers a wide range of models for natural language processing and generation. Weaviate seamlessly integrates with FriendliAI's APIs, allowing users to leverage FriendliAI's models directly within the Weaviate database.
+
+These integrations empower developers to build sophisticated AI-driven applications with ease.
+
+## Integrations with FriendliAI
+
+### Generative AI models for RAG
+
+![Single prompt RAG integration generates individual outputs per search result](../_includes/integration_friendliai_rag_single.png)
+
+FriendliAI's generative AI models can generate human-like text based on given prompts and contexts.
+
+[Weaviate's generative AI integration](./generative.md) enables users to perform retrieval augmented generation (RAG) directly within the Weaviate database. This combines Weaviate's efficient storage and fast retrieval capabilities with FriendliAI's generative AI models to generate personalized and context-aware responses.
+
+[FriendliAI generative AI integration page](./generative.md)
+
+## Summary
+
+These integrations enable developers to leverage FriendliAI's powerful models directly within Weaviate.
+
+In turn, they simplify the process of building AI-driven applications to speed up your development process, so that you can focus on creating innovative solutions.
```

Comment:
```suggestion
FriendliAI helps simplify the process of building AI-driven applications and speeds up your development process so that you can focus on creating innovative solutions.
```